### ISSUE TITLE
Alot of people are catching wind about this bypasser and how can valve patch this becuase i dont think its going to last very long

### ISSUE THREAD

**Issue by Gangrakar** [danielkrupinski/VAC-Bypass-Loader#42]

**Comment by Gangrakar** [comment]
Ive been in multiple lobbies where they were talking about it and for me this might get patched with the new anti cheat system

**Comment by rafalohaki** [comment]
pointless issue

**Comment by Robby-Sodhi** [comment]
Do they still update vac bypass loader? its been the same for like 6 months

**Comment by techzin** [comment]
> Eles ainda atualizam o carregador de desvio de vac? tem sido o mesmo há 6 meses

I believe that it is only possible to update the Bypass, when Valve repairs the codes that have been Decompiled, then the Bypass will stop working, and when that happens it will be necessary to decompile the anticheat to update this bypass.

Until it is fixed, Bypass works xD;

**Comment by NaughtDZ** [comment]
Not really, because Valve doesn't care if their VAC still works. As CSGO's official tournaments do. Valve is all about working with other third-party competitive platforms.
In fact there are a lot of easy and even open source Cheater in places like github and unknowncheats, but most of them work very well on VAC-only servers, and a lot of the VAC generated by them actually comes from OverWatch. if only VAC had FaceitAC or other anti-cheat systems If VAC had half the power of FaceitAC or other anti-cheat systems, then CSGO's official matchmaking wouldn't be what it is.
